### PR TITLE
Generalized Dashboard Integration info

### PR DESCRIPTION
**Pull request by rcinfodesign**
devhub documentation on nft integration through the dashboard generalized so it makes sense for other features as well.

### CODE REVIEW

**Comment by pawmmm** [comment]
This still seems to focus primarily on sign NFT. I would organize like this:

Step One: Set Up your Landing Page
Work with your FIO BizDev Account Representative to create your platform's branded landing page. You will be asked to provide the following:
* List of  domains you want users to register their FCH
* Logo
* Welcome text
* TPID - so that you can receive portion of all fees paid (link to info on TPID)
* Back link to your site

Step 2
* You can link your users to the Dashboard using url /ref/{refcode_provided}
* Example landing page link to Unique one or other generic demo

Contained flow
If you want your user to perform a specific action, e.g. register FCH or sign NFT and then be redirected back to your site you can pass additional url parameters
|Action|Parameters|
|---|---|
|Register/log-in and Sign NFTs|list params|

**Comment by rcinfodesign** [comment]
@pawmmm do you need any additional parameters to sign up for a FIO Crypto Handle?  And are there any other actions currently supported (to add to the table of Parameters with the SignNFT)?

File: _docs/how-to/integrate-dashboard.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+## Dashboard User Experience
+
+Your user can expect this experience while on your FIO Dashboard landing page:
+- The user will be contained to a landing page
+- The user will be required to log-in or create account
+- The user will be required to pick existing FIO Crypto Handle to sign with or register a new one
+- Domains to show will be linked to partner_id
+- Upon picking/registering their FIO Crypto Handle, the user will be redirected to confirm the action with all relevant information pre-populated and not changeable.
+- Upon performing the cation, and if redirect_url has been provided, the user will be redirected and (optionally) encrypted payload (e.g.their NFT) will be attached.
```
Comment (pawmmm) [inline _docs/how-to/integrate-dashboard.md:48]: I don't think the return payload is supported. Unique one basically queries the FIO Chain for the hash to know who just signed it (which you may want to add to this doc)

**Review by pawmmm — CHANGES_REQUESTED** [review]

File: _docs/how-to/integrate-dashboard.md (hunk 1 above)
Comment (ericbutz) [reply _docs/how-to/integrate-dashboard.md:48]: Updated with:
- Upon performing the action, and if redirect_url has been provided, the user will be redirected back to the original site.
- The referring site can then query the API using [/get_nfts_hash](http://127.0.0.1:4000/pages/api/fio-api/#post-/get_nfts_hash) to get the FIO Crypto Handle of the signer.

**Comment by ericbutz** [comment]
Closing pull request. These changes were already made on a different branch that has been merged.

**Pull request closed without merging**